modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -636,7 +637,7 @@ private String determineTemplateVersion(String frameworkVersion) {
 
     private void setUseSwashbuckle() {
         if (isLibrary) {
-            LOGGER.warn("buildTarget is " + buildTarget.getOptValue() + " so changing default isLibrary to false ");
+            LOGGER.warn("isLibrary is true so changing default useSwashbuckle to false ");
```

Comment:
Suggestion: removing the trailing whitespace.

Reply:
fixed 4fc83c67c03a26efa4a013e527af0ca56ed11866